src/facere_sensum/facere_sensum.py

```diff
@@ -4,4 +4,94 @@
 facere-sensum: make sense of the turmoil.
 '''
 
-VERSION = '0.0.1'
+from argparse import ArgumentParser
+import datetime
+import numpy as np
+import pandas as pd
+
+VERSION = '0.0.2'
+
+def score_manual(metric):
+    '''
```

Comment:
Why function comment goes in the function instead of before the function? I see this style for the first time I think. And I don't really like it, honestly speaking. I don't feel strongly about it, it's just an opinion, so let you decide. Also like `#` style of commenting better. But it's also a personal preference.

Reply:
This is because these are not comments :) These are so called 'docstrings': a docstring is a string literal that occurs as the first statement in a module, function, class, or method definition. Such a docstring becomes the __doc__ special attribute of that object.
As you can imagine, there are automated tools that process docstrings, so it is important to have them this way.